src/VisualStudio/Core/Def.Next/project.json

```diff
@@ -0,0 +1,18 @@
+﻿{
+  "dependencies": {
+    "ManagedEsent": "1.9.2.0",
+    "Microsoft.CodeAnalysis.Elfie": "0.10.6-rc2",
+    "Microsoft.VisualStudio.Shell.15.0-pre": "15.0.25511.2-pre",
+    "Microsoft.VisualStudio.Shell.Framework-pre": "15.0.25511-pre",
+    "NuGet.VisualStudio": {
+      "version": "3.3.0",
+      "suppressParent": "all"
+    },
+    "RoslynDependencies.Microsoft.VisualStudio.CallHierarchy.Package.Definitions": "14.3.25407",
+    "RoslynDependencies.Microsoft.VisualStudio.GraphModel": "14.3.25407",
+    "RoslynDependencies.Microsoft.VisualStudio.Progression": "14.3.25407"
```

Comment:
You shouldn't need all of these, because they should flow from on of the other VisualStudio projedcts.